compose/service/record.go

```diff
@@ -537,6 +536,12 @@ func (svc record) Bulk(oo ...*types.BulkRecordOperation) (rr types.RecordSet, er
 		return nil
 	})
 
+	// no need for another round of action recording, since it was already
```

Comment:
What is recorded here is the fact that this was a bulk op. So, let's loose this check.